src/streamlink/plugins/pluto.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from uuid import uuid4
+
+from streamlink.plugin import Plugin
+from streamlink.stream import HLSStream
+
+
+class Pluto(Plugin):
+    url_re = re.compile(r'https?://(?:www\.)?pluto\.tv/live-tv/(?P<slug>[a-zA-Z0-9-]+)')
+
+    api_url = 'https://api.pluto.tv/v2/channels'
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls.url_re.match(url) is not None
+
+    def _get_streams(self):
+        slug = self.url_re.match(self.url)['slug'].lower()
+
+        channels_res = self.session.http.get(self.api_url)
+        channels_data = channels_res.json()
+        channel_match = next(filter(lambda x: x['slug'] == slug, channels_data), None)
+
+        if not channel_match:
+            self.logger.error('Channel %s not found. It may have been removed.' % slug)
```

Comment:
You should use the standard python logging api, the `plugin.logger` api has been deprecated.

```
import logging
log = loggering.getLogger(__name__)

...

log.error(....)
```